clients/python-wrapper/lakefs/object_io.py

```diff
@@ -221,29 +222,66 @@ def _extract_etag_from_response(headers) -> str:
         etag = headers.get("ETag", "").strip(' "')
         return etag
 
-    def _upload(self, content, pre_sign, content_type: Optional[str] = None,
-                metadata: Optional[dict[str, str]] = None) -> ObjectStats:
-        if not pre_sign:
-            raise UnsupportedOperationException("Upload currently supported only in pre-sign mode")
-
-        headers = {}
-        if content_type is not None:
-            headers["Content-Type"] = content_type
+    def _upload_raw(self, content: UploadContentType, content_type: Optional[str] = None,
+                    metadata: Optional[dict[str, str]] = None) -> ObjectStats:
+        """
+        Use raw upload API call to bypass validation of content parameter
+        """
+        headers = {
+            "Accept": "application/json",
+            "Content-Type": content_type if content_type is not None else "application/octet-stream"
+        }
+
+        # Create user metadata headers
+        for k, v in metadata.items():
+            headers[_LAKEFS_METADATA_PREFIX + k] = v
+
+        _response_types_map = {
+            '201': "ObjectStats",
+            '400': "Error",
+            '401': "Error",
+            '403': "Error",
+            '404': "Error",
+            '412': "Error",
+            '420': None,
```

Comment:
Should we add a TODO to support "enhance your calm"?  And until we do, isn't it a ServerError?

Reply:
Not sure what you mean, but I think ServerError is the fallback

Reply:
The map was copied 1:1 from the SDK code

Reply:
I know.  We shouldn't copy stuff that gets auto-generated.